cleanlab/filter.py

```diff
@@ -310,83 +314,63 @@ class 0, 1, ..., K-1.
             prune_count_matrix = round_preserving_row_totals(tmp)
 
         # Prepare multiprocessing shared data
-        if n_jobs > 1:
-            _labels = RawArray("I", labels)  # type: ignore
-            _label_counts = RawArray("I", label_counts)  # type: ignore
-            _prune_count_matrix = RawArray("I", prune_count_matrix.flatten())  # type: ignore
-            _pred_probs = RawArray("f", pred_probs.flatten())  # type: ignore
-        else:  # Multiprocessing is turned off. Create tuple with all parameters
-            args = (
-                labels,
-                label_counts,
-                prune_count_matrix,
-                pred_probs,
-                multi_label,
-                min_examples_per_class,
-            )
+        chunksize = max(1, K // n_jobs)
+        os_name = platform.system()
```

Comment:
could you add brief comment here explaining what will differ based on operating system type?